changelogs/changelog.yaml

```diff
@@ -1 +1,23 @@
-releases: {}
+ancestor: null
+releases:
+  1.1.0:
+    changes:
+      deprecated_features:
+      - All community.azure.azure_rm_<resource>_facts modules are deprecated. Use azure.azcollection.azure_rm_<resource>_info
+        modules instead (https://github.com/ansible-collections/community.azure/pull/24).
+      - All community.azure.azure_rm_<resource>_info modules are deprecated. Use azure.azcollection.azure_rm_<resource>_info
+        modules instead (https://github.com/ansible-collections/community.azure/pull/24).
+      - community.azure.azure_rm_managed_disk is deprecated. Use azure.azcollection.azure_rm_managed_disk
```

Comment:
```suggestion
      - community.azure.azure_rm_managed_disk is deprecated. Use azure.azcollection.azure_rm_manageddisk
```

Reply:
Also, what about azure_rm_manageddisk in this collection? Is this not going to be deprecated?

Reply:
Good catch about ``azure_rm_managed_disk`` -> ``azure_rm_manageddisk``, I knew that but it must have escaped me.

Same for ``community.azure.azure_rm_manageddisk``, it should have been deprecated as well but I must have missed it when proposing the patch.

Thanks for the review, should be fixed in https://github.com/ansible-collections/community.azure/pull/24/commits/b91dc276a0d8864884a8c080f8a067bfbac35d58